### DISCUSSION TITLE
Urgent Inquiry Regarding Your Product Edition (Community and Enterprise)

### DISCUSSION THREAD

**Discussion by zulu093** [gravitational/teleport#40078] [Q&A] [answered]
Hello, we have been utilizing versions 14 through 15 of Teleport Community Edition for server account management.
But we found the notice that version 16 may require a payment, and please answer below questions:

1. Have we use a paid Enterprise version even we are currently useing Teleport Community Edition versions 14 and 15?
2. We are planning to connect up to 200 servers; can this be accommodated within the Community Edition?
3. Do you have a customer or sales contact numer or email in Korea?

We are looking for your response, thank you.

**Comment by webvictim** [comment, marked as answer]
Hello! The best way to get answers to these questions is to email sales@goteleport.com with your questions and contact details. We will reply to with detailed answers.

Thank you!
